src/services/auth/index.tsx

```diff
@@ -172,5 +191,95 @@ export const mockValidateOTP = async (
   };
 };
 
+/** Error message returned by endpoints without a /auth/logout handler */
+const oldBackendError =
+  "Authorization header requires 'Credential' parameter. Authorization header requires 'Signature' parameter. Authorization header requires 'SignedHeaders' parameter. Authorization header requires existence of either a 'X-Amz-Date' or a 'Date' header. Authorization=";
+
+export const liveCallLogout = async (
+  sessionToken: string,
+  operatorToken: string,
+  endpoint: string
+): Promise<void> => {
+  const payload = { sessionToken, operatorToken };
+
+  try {
+    await fetchWithValidator(LogoutResponse, `${endpoint}/auth/logout`, {
+      method: "POST",
+      headers: {
+        Authorization: sessionToken,
+      },
+      body: JSON.stringify({
+        operatorToken,
+      }),
+    });
+    Sentry.addBreadcrumb({
+      category: "logout",
+      message: JSON.stringify({
+        status: "success",
+        ...payload,
+        endpoint,
+      }),
+    });
+    return;
+  } catch (e) {
+    if (e instanceof NetworkError) {
+      throw e;
+    } else if (
+      e instanceof SessionError ||
+      e.message === "Auth token is not currently valid" ||
+      e.message === oldBackendError + sessionToken
+    ) {
+      // for this case the session token is already invalidated or past the valid date
+      // logoutTodo: is it safe to keep session token there if already past validity date
+
+      Sentry.addBreadcrumb({
+        category: "logout",
+        message: JSON.stringify({
+          status: "success despite error",
```

Comment:
Wait what is success despite error I'm confused 😅

Reply:
Error is thrown by the backend, but we can still consider it as a successful logout.
E.g. if the sessionToken is already non-existent on the BE, it will throw an error. In this case the user does not need to do any further action (because the sessionToken already doesn't work anyway), so we just silently discard the sessionToken on the FE side.

Reply:
Can we describe it better in the status message / a comment?